include/linux/soundwire/sdw.h

```diff
@@ -683,6 +693,15 @@ struct sdw_slave {
 	struct completion initialization_complete;
 	u32 unattach_request;
 	bool first_interrupt_done;
+
+	u32 rw_transactions;
+	u32 read_transactions;
+	u32 write_transactions;
+	u32 paged_transactions;
+	u32 paged_transactions_local;
+	u32 paged_transactions_nonlocal;
```

Comment:
the latter two are simple counters, so `unsigned int` should be fine for them.

Reply:
This counts the transactions since the initial probe, I wasn't sure how many there would be. 
But doing a bit of math, the worst case is similar to audio, the command rate is 48kHz, so the counters would wrap-around in 24.85 hours with a regular 32-bit counter. 
That's worst case, assuming a command per frame, which is practice is not the case at all. So I think indeed unsigned int is more than enough.